### DISCUSSION TITLE
where can I find the free canva pro for student?

### DISCUSSION THREAD

**Discussion by Mayearugay** [community/community#28014] [GitHub Education] [answered]
where can I find the free canva pro for student?

**Comment by truongletann** [comment]
you just need to have the school email you can get to register canva for students. You can see how to do it here: https://www.youtube.com/watch?v=p6ZK557tCUE

**Comment by samus-aran** [comment, marked as answer]
Hi @Mayearugay  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers). For students who have already claimed for their 12 month benefit, you will continue to have access for the remainder of your 12 month offer period and can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.
